examples/gitserver/gitserver.yaml

```diff
@@ -70,12 +70,17 @@ items:
           - name: HOOK_PATH
             value: /var/lib/git-hooks
 
-          # The script to use to do custom language detection on a 
+          # If 'yes' new-app will be invoked on push for repositories
+          # for which a matching BuildConfig is not found.
+          - name: GENERATE_ARTIFACTS
+            value: "yes"
```

Comment:
I don't have an issue with changing these flags to whatever is consistent.

On Tue, Apr 19, 2016 at 6:21 PM, Ben Parees <notifications@github.com>
wrote:

> In examples/gitserver/gitserver.yaml
> <https://github.com/openshift/origin/pull/8394#discussion_r60322525>:
>
> > @@ -70,12 +70,17 @@ items:
> >            - name: HOOK_PATH
> >              value: /var/lib/git-hooks
> >
> > -          # The script to use to do custom language detection on a
> > +          # If 'yes' new-app will be invoked on push for repositories
> > +          # for which a matching BuildConfig is not found.
> > +          - name: GENERATE_ARTIFACTS
> > +            value: "yes"
>
> unless @smarterclayton <https://github.com/smarterclayton> wants to
> defend the decision i'd like to change it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8394/files/b753c78d158936df66b3ee42e706145414da118b..1b2dc0730486fb06774d4b77e49e4b3990d007d2#r60322525>
>